android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1940,6 +2058,34 @@ private List<String> getStringArrayList(@Nullable ReadableArray readableArray)
         return list;
     }
 
+    private WritableArray getWritableArray(@Nullable List<String> list)
```

Comment:
Since you nullability marked the param, you should mark return type as such as well. Also, missing `final`.

Reply:
Fixed to return @Nullable.